APILambda/test/userFlows/events.ts

```diff
@@ -34,12 +35,13 @@ export const createEventFlow = async (
 
   for (let i = 0; i < attendeeCount; i++) {
     const attendee = await createUserFlow()
-
     attendeesList.push(attendee)
 
-    await Promise.all(
-      eventIds.map((eventId) => callJoinEvent(attendee.token, eventId))
-    )
+    for (const eventId of eventIds) {
+      // delay needed for test attendees to be properly sorted by join time
+      await delay(1000)
+      await callJoinEvent(attendee.token, eventId)
+    }
```

Comment:
We could load the DB with some preset users at the start of this test by specifying their join time manually. Though I get this is supposed to be as "black-box" as possible. If you want to keep the black box aspect, then you could lower the delay to 10ms.